include/benchmark/benchmark.h

```diff
@@ -153,14 +154,18 @@ void Initialize(int* argc, const char** argv);
 
 // Otherwise, run all benchmarks specified by the --benchmark_filter flag,
 // and exit after running the benchmarks.
-void RunSpecifiedBenchmarks(const BenchmarkReporter* reporter = nullptr);
+void RunSpecifiedBenchmarks(const BenchmarkReporter* reporter = NULL);
 
 // ------------------------------------------------------
 // Routines that can be called from within a benchmark
 
 //
 // REQUIRES: a benchmark is currently executing
-void SetLabel(const std::string& label);
+void SetLabel(const char* label);
```

Comment:
please remove these in favour of setting it on 'state' for consistency with everything else.